app/models/project.rb

```diff
@@ -14,10 +14,14 @@ class Project < ApplicationRecord
   has_many :user_actions
   has_many :data_requests, as: :exportable
 
-  enum status: { paused: 0, active: 1 }
+  enum status: { halted: 0, active: 1 }
 
   attr_accessor :rerun
 
+  def paused?
```

Comment:
so project can't be paused anymore, is it worth keeping this around at all? I can't see in this PR that it's still in use so i'd vote for removal. 

If it's needed for backwards compat with existing code (that can't be changed) then we could proxy to the halted state enum value vs renaming all uses?

Reply:
this method is here so that i don't have to check on a reducible's type before checking to see if it's paused. projects can't be paused according to the new definition of paused because they don't have extractors. but it's nice to just check `reducible.paused?` instead of `reducible is Workflow && reducible.paused?`

Reply:
> projects can't be paused according to the new definition of paused because they don't have extractors.

ok - mind adding a comment about the logic behind the project not allowed to be paused?

Reply:
sure!